jwt/jwt.c

```diff
@@ -83,7 +83,18 @@ char* jwt_create(char * project_id, long long int time, NN_DIGIT * priv_key, int
 
 int jwt_verify(char * jwt,  point_t * pub_key)
 {
-    ecc_init(); // TODO: Has to be initialized only once -> move to broker init phase
+    //ecc_init(); // TODO: Has to be initialized only once -> move to broker init phase
+
+    int dot_count = 0;
+    for (char *ptr = jwt; *ptr != '\0'; ptr++) {
+        if (*ptr == '.') {
+            dot_count++;
+        }
+    }
+    if (dot_count < 2) {
+        fprintf(stderr, "Invalid JWT: Too few segments\n");
```

Comment:
it's not an error; it's an invalid user input. Do not print it as error. And again... use mosquitto logging system if you want to log something.